src/main/java/mate/academy/model/User.java

```diff
@@ -49,4 +57,12 @@ public void setPassword(String password) {
     public void setSalt(byte[] salt) {
         this.salt = salt;
     }
+
+    @Override
+    public String toString() {
+        return "User{"
+                + "email='" + email + '\''
+                + ", password='" + password + '\''
```

Comment:
it's better to remove password from User.toString()

Reply:
F..k, misscliked, thanks=)